openapi/Swarm.yaml

```diff
@@ -861,11 +861,107 @@ paths:
               schema:
                 $ref: 'SwarmCommon.yaml#/components/schemas/ReferenceResponse'
         '400':
-           $ref: 'SwarmCommon.yaml#/components/responses/400'
+          $ref: 'SwarmCommon.yaml#/components/responses/400'
         '401':
-           $ref: 'SwarmCommon.yaml#/components/responses/401'
+          $ref: 'SwarmCommon.yaml#/components/responses/401'
         '500':
            $ref: 'SwarmCommon.yaml#/components/responses/500'
         default:
           description: Default response
 
+  '/feed/{owner}/{topic}':
```

Comment:
Signature seems to be missing.

Reply:
not needed for `POST` since no SOC is being created